july12/index.html

```diff
@@ -297,6 +298,12 @@
 
     window.optimizely || (window.optimizely = []);
 
+    var params = new URLSearchParams(window.location.search.substring(1));
+    if (params.get('org') == 'dp') {
+      global.demandProgress = true;
```

Comment:
Is this being used anywhere? Would prefer to avoid adding anything more to global namesapce unless absolutely necessary.